build.gradle

```diff
@@ -30,16 +30,16 @@ buildscript {
 }
 
 plugins {
-  id 'com.diffplug.spotless' version '6.10.0'
-  id 'com.github.ben-manes.versions' version '0.42.0'
+  id 'com.diffplug.spotless' version '6.14.1'
```

Comment:
The latest version of `spotless-plugin-gradle` in maven is `6.14.0`.
ref: https://mvnrepository.com/artifact/com.diffplug.spotless/spotless-plugin-gradle

Reply:
6.14.1 is available, see here https://search.maven.org/artifact/com.diffplug.spotless/spotless-plugin-gradle/6.14.1/jar

But it requires Java 11 to build, so I had to downgrade to 6.13.0..

I'll ping reviewers once I get a green build. :)